frontend/src/components/FAQ/faq.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./styles/style.css";
+import "./styles/style.scss";
```

Comment:
We aren't using `scss` anywhere so in order to be consistent let's go only  with `css`

Reply:
ok, I removed scss file